### PR TITLE
Set the correct flag when init with a value

### PR DESCRIPTION
**Pull request by mimse**

### CODE REVIEW

**Review by ManBearTM — APPROVED** [review]
Would it make sense to add a test for this functionality?

**Comment by mimse** [comment]
> Would it make sense to add a test for this functionality?

It would, I will make a task for it in the bounty board

File: libs/watt/src/lib/components/phone-field/watt-phone-field.component.ts
Comment (dzhavat) [inline libs/watt/src/lib/components/phone-field/watt-phone-field.component.ts:1]: praise: the flag appears correctly now. Good job 💯

issue: when opening the profile modal for a second time, a console error is thrown about `metadata` argument.

bug

Loading the  `metadata` eagerly seems to fix the issue. Does loading the `metadata` eagerly affect the final bundle a lot?

Comment (dzhavat) [inline libs/watt/src/lib/components/phone-field/watt-phone-field.component.ts:1]: issue:: the phone is not formatted correctly when opening the modal initially. Eagerly loading the  `metadata` seems to fix that (on opening the profile modal for the second time).

bug2

**Pull request closed without merging**